docs/how-to/command-line-quick-start.md

```diff
@@ -6,117 +6,149 @@ description: Quick-start guide for installing and getting started with IPFS from
 
 # Command-line quick start
```

Comment:
Can you make sure this is ALSO the first thing in the install section? This is the best explanation for how to install IPFS - so should be easily findable there

Reply:
The [Command-line quick start](https://docs-beta.ipfs.io/how-to/command-line-quick-start/#install-ipfs) is already linked to at the top of [`docs-beta.ipfs.io/install/`](https://docs-beta.ipfs.io/install/)

```
# Install IPFS

We're glad you're interested in giving IPFS a try! There are a lot of different ways to install and use IPFS — the best way to find the right one for you is first to consider what you want to accomplish or try out with IPFS. Take a look at the guide below to find the flavor of IPFS that's best for your needs.

## Quick-start

Are you a developer looking to test-drive IPFS for your needs? Follow our [quick-start guide](/how-to/command-line-quick-start) to install our reference implementation and get up and running ASAP!

## Integrate IPFS into your app
```

Is this alright? Or are you referring to something else?

Reply:
I mean in the sidenav I'd expect to see it in the Install section (instead of hidden in the "how tos")

Reply:
Ah right! I see what you mean. 5b0039c addresses this issue. The `Command-line quick start` page is now found in both `/install` and `/how-tos`. This is so that we don't break any inbound links coming from other places. We can fix this stuff with a re-direct _after_ the 0.5 launch.